content/developer/reference/backend/views.rst

```diff
@@ -1863,6 +1863,81 @@ Possible children elements of the list view are:
 
   A special button (`type="edit"`) can be defined to open the many2one form view.
 
+``header``
+  defines custom buttons in the control panel that performs an action/calls a model's method.
+
+  .. code-block:: xml
+
+      <header>
+          <button name=toDo" type="object" string="Always Display" display="always"/>
+          <button name="toDo" type="object" string="Selection Display"/>
+      </header>
+
+  ``button``
+    displays a button in a list cell
+
+    ``icon``
+        icon to use to display the button
+    ``string``
+        * if there is no ``icon``, the button's text
+        * if there is an ``icon``, ``alt`` text for the icon
+    ``type``
+        type of button, indicates how it clicking it affects Odoo:
+
+        ``object``
+            call a method on the list's model. The button's ``name`` is the
+            method, which is called with the current row's record id and the
+            current context.
+
+            .. web client also supports a @args, which allows providing
+               additional arguments as JSON. Should that be documented? Does
+               not seem to be used anywhere
+
+        ``action``
+            load an execute an ``ir.actions``, the button's ``name`` is the
+            database id of the action. The context is expanded with the list's
+            model (as ``active_model``), the current row's record
+            (``active_id``) and all the records currently loaded in the list
+            (``active_ids``, may be just a subset of the database records
+            matching the current search)
+
+    ``display``
+      attribute allowing to know when the button should be displayed.
+      The default value is  ``selection``.
+        ``always``
+          the button is always displayed.
+        ``selection``
+          the button is only displayed if at least one record is selected.
```

Comment:
By default, those buttons are only displayed when some records are selected, and they apply on the selection. When the attribute `display` is set to `always`, the button is available all the time, even if there's no selection.